src/Core/NuGet/Core/Package.nuspec

```diff
@@ -11,6 +11,7 @@
     <iconUrl>$image$</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>Application Insights Core API. This package provides core functionality for transmission of all Application Insights Telemetry Types and is a dependent package for all other Application Insights packages. Please install the platform specific package for the best experience.</description>
+    <summary>Application Insights Core API. This package provides core functionality for transmission of all Application Insights Telemetry Types and is a dependent package for all other Application Insights packages. Please install the platform specific package for the best experience. Privacy statement: https://go.microsoft.com/fwlink/?LinkId=512156</summary>
```

Comment:
Should the privacy link go in the description tag instead of the summary tag?  Or will it not get displayed if both the summary and description tags exist?